docker-compose.yml

```diff
@@ -3,8 +3,7 @@ version: '3.4'
 services:
 
   lpvs:
-    build:
-      context: .
```

Comment:
I think we can leave these for `docker compose build` command.

Reply:
I'll check